spec/api/v1/projects_spec.rb

```diff
@@ -0,0 +1,320 @@
+require 'rails_helper'
+
+RSpec.describe V1::Projects do
+  let(:api_token) { create :api_token }
+
+  before do
+    Timecop.freeze(Time.local(2016, 12, 7, 10, 0))
+  end
+
+  after do
+    Timecop.return
+  end
+
+  describe '#GET /api/v1/projects' do
+    let(:team) { create :team }
+    subject { get '/api/v1/projects', per_page: 2, api_key: api_token.token }
+
+    before do
+      create_list :project, 4, teams: [team]
+    end
+
+    it 'returns 2 projects' do
+      subject
```

Comment:
the same, here, it makes the test easier to read, or if you add a name to the subject, I'm ok too